app/models/dimensions/item.rb

```diff
@@ -5,6 +5,10 @@ class Dimensions::Item < ApplicationRecord
 
   validates :content_id, presence: true
 
+  def self.by_natural_key(content_id:, locale:)
```

Comment:
I know it is already been covered in the integration tests, but could you please add a unit test here?

Reply:
It already has one ```describe '##by_natural_key' do``` mentioned below